dbms/src/Common/filesystemHelpers.cpp

```diff
@@ -0,0 +1,27 @@
+#include <Common/filesystemHelpers.h>
+#include <common/config_common.h>
+#include <Poco/File.h>
+#include <Poco/Path.h>
+
+namespace DB
+{
+
+bool checkFreeSpace(const std::string & path, size_t data_size)
+{
+#if !UNBUNDLED
```

Comment:
Why can't we get free space if we're built in unbundled mode?

Reply:
I don't know. It's not related to this patch. It just keep logic as is.